config/environments/development.rb

```diff
@@ -19,7 +19,7 @@
   config.domain = ENV["DOMAIN"]
   config.gias_api_root = ENV["GIAS_API_ROOT"]
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
```

Comment:
This will now tell us when an email hasn't been sent because of a "dodgy" email address or we can a failed to send response?

Reply:
This will tell us if Notify refuses to send our email. This would happen if we tried to send something with a test api key, to someone outside of our tam and whitelist.